src/MessageContainer.js

```diff
@@ -135,10 +135,20 @@ export default class MessageContainer extends React.Component {
     );
   }
 
+  endReachedProps(){
+    let condProps = {}
+    if (this.props.onTopReached){ condProps.onEndReached = this.props.onTopReached }
```

Comment:
Sorry if I'm a little bit confused but, Are you using `onEndReached` to know if you're on top of list?

Maybe you wanted to use [`onScroll`](https://facebook.github.io/react-native/docs/scrollview.html#onscroll) property?